Lib/test/test_importlib/import_/test_api.py

```diff
@@ -96,6 +96,20 @@ def test_blocked_fromlist(self):
                                     fromlist=[SUBMOD_NAME.rpartition('.')[-1]])
                 self.assertEqual(cm.exception.name, SUBMOD_NAME)
 
+    def test_fromlist_invalid_type(self):
+        with self.assertRaises(TypeError) as cm:
+            self.__import__('encodings', fromlist=[b'aliases'])
+        self.assertEqual(
+            str(cm.exception),
+            "Sequence item 0 in 'from list' must be str, not 'bytes'"
+        )
+        with self.assertRaises(TypeError) as cm:
+            self.__import__('encodings', fromlist=['aliases', b'codecs'])
```

Comment:
I think it is better to use 'utf_8' instead of 'codecs'. 'codecs' is just an outer module imported  in 'encodings'. It could be imported as '_codecs' for example.